concordia/views.py

```diff
@@ -1128,17 +1129,34 @@ def reserve_asset(request, *, asset_pk):
 
     timestamp = now()
 
-    # First clear old reservations, with a grace period:
+    # First clear old reservations, with a grace period.
     cutoff = timestamp - (
         timedelta(seconds=2 * settings.TRANSCRIPTION_RESERVATION_SECONDS)
     )
 
+    reservations = []
+
     with connection.cursor() as cursor:
-        cursor.execute(
-            "DELETE FROM concordia_assettranscriptionreservation WHERE updated_on < %s",
+        rows_to_release = cursor.execute(
+            (
+                "DELETE FROM concordia_assettranscriptionreservation "
+                "WHERE updated_on < %s "
+                "RETURNING asset_id as asset_pk, user_id as user_pk"
+            ),
             [cutoff],
         )
 
+    if rows_to_release:
+        for row in rows_to_release.fetchone():
```

Comment:
Shouldn't this be `fetchall`?

```suggestion
        for row in rows_to_release.fetchall():
```